src/Tizen.Multimedia.Vision/MediaVision/InferenceType.cs

```diff
@@ -37,7 +37,24 @@ public enum InferenceBackendType
         /// <summary>
         /// ArmNN backend type
         /// </summary>
-        ArmNN
+        ArmNN,
+
+        /// <summary>
+        /// ML Single API of NNStreamer backend type
+        /// </summary>
+        /// <remarks>
+        /// Should be set <see cref="InferenceTargetDevice"/> to <see cref="InferenceTargetDevice.Custom"/>.
+        /// </remarks>
+        /// <since_tizen> 8 </since_tizen>
+        MLApi,
+
+        /// <summary>
+        /// On-device Neural Engine backend type
+        /// </summary>
+        /// Should be set <see cref="InferenceTargetDevice"/> to <see cref="InferenceTargetDevice.CPU"/> or
+        /// <see cref="InferenceTargetDevice.GPU"/>.
```

Comment:
Missing `<remarks>` tags.
```suggestion
         /// <remarks>
        /// Should be set <see cref="InferenceTargetDevice"/> to <see cref="InferenceTargetDevice.CPU"/> or
        /// <see cref="InferenceTargetDevice.GPU"/>.
        /// </remarks>
```

Reply:
Okay. I missed `<ramarks>` tag and it'll be fixed.